mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/English.kt

```diff
@@ -38,16 +48,16 @@ object English : LocalizedStrings {
   override val sectionSocial = "Players"
   override val sectionSettings = "Settings"
   override val sectionPlay = "Play"
-
-  override val boardColorBlack = "black"
-  override val boardColorWhite = "white"
-  override val boardPieceKing = "king"
-  override val boardPieceQueen = "queen"
-  override val boardPieceRook = "rook"
-  override val boardPieceBishop = "bishop"
-  override val boardPieceKnight = "knight"
-  override val boardPiecePawn = "pawn"
-  override val boardPieceContentDescription = { color: String, rank: String -> "$color $rank" }
-
   override val newGame = "New game".uppercase()
+
+  override val pregameChooseColor = "Pick your color :"
+  override val pregameChooseOpponent = "Pick an opponent :"
+  override val pregameChooseGame = "Pick your game :"
+  override val pregameWhiteColor = "White".uppercase()
+  override val pregameBlackColor = "Black".uppercase()
+  override val pregameStart = "Start!".uppercase()
+  override val pregameSelectOpponent = "Select".uppercase()
+  override val pregameSelectedOpponent = "Selected".uppercase()
+  override val pregamePlayLocal = "Play locally".uppercase()
+  override val pregamePlayOnline = "Play against opponent".uppercase()
```

Comment:
Suggestion : I think naming these `prepareGameXXX` is probably more explicit.

Reply:
Agree !